webpack.config.js

```diff
@@ -65,15 +66,15 @@ module.exports = function (env, argv) {
         template: './src/options/index.html',
         chunks: ['options'],
       }),
-      ...(isProd
+      ...(isProd || !dotEnv.CHROMIUM_BINARY
```

Comment:
There is no reason for skipping this, if you don't want to start a browser then you can simply run `build:chromium`.

Reply:
two reasons:

1. development build is faster and is different from production build. build:chromium is production build.
2. and, production build doesn't supports watch.

I will still suggest to handle this problem, otherwise every developers without chromium in their environment will encounter slower development speed.

Reply:
https://github.com/tongwentang/tongwentang-extension/issues/26#issuecomment-821986613